### PR TITLE
Fix attribute mapping issues

### PR DESCRIPTION
**Pull request by JayaShakthi97**
### Purpose
- Fix attribute mapping issue

### Related Issues
- None

### Checklist
- [ ] e2e cypress tests locally verified.
- [x] Manual test round performed and verified.
- [ ] UX/UI review done on final implementation.
- [ ] Documentation provided. (Add links if there's any)
- [ ] Unit tests provided. (Add links if there's any)
- [ ] Integration tests provided. (Add links if there's any)

### Related PRs
- https://github.com/wso2/identity-apps/pull/2970

### Security checks
- [ ] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [ ] Ran FindSecurityBugs plugin and verified report?
- [ ] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

File: apps/console/src/features/claims/constants/claim-management-constants.ts [hunk 1]
```diff
@@ -119,7 +119,12 @@ export class ClaimManagementConstants {
         SCIMConfigs.oidc
     ];
 
-    public static readonly SCIM_MAPPING: string[] = Object.values(SCIMConfigs.scim);
+    public static readonly SCIM_MAPPING: string[] = [
```
Comment (brionmario) [inline apps/console/src/features/claims/constants/claim-management-constants.ts:122]: Any reason for duplicating this?
`SCIMConfigs.scim` seems to have the values.
Can't we use `Object.values(SCIMConfigs.scim)`?

Comment (JayaShakthi97) [reply apps/console/src/features/claims/constants/claim-management-constants.ts:122]: `SCIM config` file is used to map scim attributes.
`claim-management-constants` file is used for rendering.
Values of these two files can differ in different usecases. Better to have in two files.

Comment (JayaShakthi97) [reply apps/console/src/features/claims/constants/claim-management-constants.ts]: Fixed with e77c40b16a7ebe4406558279bd52cfd5ac39181e

File: apps/console/src/features/claims/components/edit/external-dialect/edit-external-claim.tsx [hunk 2]
```diff
@@ -324,13 +324,14 @@ export const EditExternalClaim: FunctionComponent<EditExternalClaimsPropsInterfa
...
-                                            </Code>
-                                        </div>),
+                                                { claim?.displayName }
+                                                <Code 
+                                                    className="description" 
+                                                    compact 
+                                                    withBackground={ false }>
+                                                    { claim?.claimURI }
+                                                </Code>
+                                            </div>),
```
Comment (brionmario) [inline apps/console/src/features/claims/components/edit/external-dialect/edit-external-claim.tsx:334]:
```suggestion
                                            </div>
                                        ),
```

**Review by brionmario — APPROVED** [review]

**Pull request merged**